src/components/IndexTable/IndexTable.tsx

```diff
@@ -521,6 +553,7 @@ function IndexTableBase({
     hasMoreLeftColumns && styles['Table-scrolling'],
     selectMode && styles.disableTextSelection,
     selectMode && shouldShowBulkActions && styles.selectMode,
+    isSelectableIndex === false && styles['Table-unselectable'],
```

Comment:
[nit] for consistency with how it's used elsewhere, [c/sh]ould this be `!isSelectableIndex`?

Reply:
Yup, updated, also renamed `isSelectableIndex` to `selectable` for better readability